api/src/pcapi/core/offers/models.py

```diff
@@ -1438,3 +1438,12 @@ class OfferCompliance(PcObject, Base, Model):
     compliance_reasons: sa_orm.Mapped[list[str]] = sa.Column(
         MutableList.as_mutable(postgresql.ARRAY(sa.String)), nullable=False
     )
+
+
+class FutureOfferReminder(PcObject, Base, Model):
+    __tablename__ = "future_offer_reminder"
+
+    user: sa.orm.Mapped["User"] = sa.orm.relationship("User", backref="future_offer_reminders")
+    user_id: int = sa.Column(sa.BigInteger, sa.ForeignKey("user.id"), index=True, nullable=False)
```

Comment:
c'est pas fou, mais on a l'habitude du camelCase pour les id des FK

```suggestion
    userId: int = sa.Column(sa.BigInteger, sa.ForeignKey("user.id"), index=True, nullable=False)
```

Reply:
Toutes les colonnes en db sont en camelCase, c'est horrible mais c'est la vie

Reply:
oui c'est ce que je voulais dire en fait 😇